### PR TITLE
Add loongarch64 support

### PR DESCRIPTION
**Pull request by yzewei**
The LoongArch architecture (LoongArch) is an Instruction Set Architecture (ISA) that has a RISC style.

### CODE REVIEW

**Comment by FedeDP** [comment]
Hi! Thanks for opening this PR! Are you planning to add support for loongarch to drivers?

**Comment by yzewei** [comment]
> Hi! Thanks for opening this PR! Are you planning to add support for loongarch to drivers?

This is what I want to do

**Comment by FedeDP** [comment]
Yay! Thanks! Keeping this on hold until Pes get merged on libs then ;)

**Comment by yzewei** [comment]
> Yay! Thanks! Keeping this on hold until PRs get merged on libs then ;)

There seems to be no problem.

**Comment by FedeDP** [comment]
Closing and reopening to trigger CI!
/close

**Comment by FedeDP** [comment]
@yzewei can you rebase this one on latest main?

**Review by FedeDP — APPROVED** [review]
/approve

**Comment by yzewei** [comment]
> can you rebase this one on latest main?

@FedeDP Sorry, I just saw it.

**Comment by FedeDP** [comment]
No problem :)

**Pull request merged**
